seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -549,7 +549,7 @@
         "Here's how you would rewrite and italicize the text of our heading:",
```

Comment:
There is a second instance of the word italicize here.

You can update your commit by doing `git commit --amend`.